src/context/provider.js

```diff
@@ -9,7 +9,8 @@
  *             to next one, for example data about material to track when it is changed.
  * @notes   Should hold static data only (see `setData` method), no classes or functions
  */
-import lodash from "lodash";
+import capitalize from "lodash/capitalize";
```

Comment:
I understand where you're coming from on this, but I have to go with readability on this: `lodash.get` is way more understandable than `get` when reading the code. Plus, the 150.5KB size is not as important for us, given that the bundle size is measured in MBs. You can put a separate task for it, if you'd like.

Reply:
ok no problem, I can change it back